Mathlib/Algebra/Lie/Killing.lean

```diff
@@ -592,6 +592,47 @@ lemma ker_weight_inf_rootSpaceProductNegSelf_eq_bot [CharZero K] (α : weight K
 
 end IsKilling
 
+section LieAlgebraEquiv
+
+variable {R L L' : Type*} [Field R]
+  [LieRing L] [LieAlgebra R L]
+  [LieRing L'] [LieAlgebra R L']
+
+/-- Given an equivalence `e` of Lie algebras from `L` to `L'`, and elements `x y : L`, the
+respective Killing forms of `L` and `L'` satisfy `κ'(e x, e y) = κ(x, y)`. -/
+lemma killingForm_of_equiv_apply (e : L ≃ₗ⁅R⁆ L') (x y : L) :
+    killingForm R L' (e x) (e y) = killingForm R L x y := by
+  simp_rw [LieModule.traceForm_apply_apply]
+  rw [← ad, ← ad]
+  simp_rw [← LieAlgebra.conj_ad_apply]
+  rw [← LinearEquiv.conj_comp]
+  rw [LinearMap.trace_conj']
+
+/-- Given a Killing Lie algebra `L`, if `L'` is isomorphic to `L`, then `L'` is Killing too. -/
+lemma isKilling_of_equiv [h : IsKilling R L] (e : L ≃ₗ⁅R⁆ L') : IsKilling R L' := by
```

Comment:
Please create an alias called `LieAlgebraEquiv.isKilling` which can be called with dot-notation.
```
alias LieAlgebraEquiv.isKilling := isKilling_of_equiv
```

Reply:
Yes. I actually added `alias _root_.LieEquiv.isKilling := LieAlgebra.isKilling_of_equiv` to be in the correct namespace so that I could indeed use dot-notation to call it. I also renamed the section to `LieEquiv` (since `LieAlgebraEquiv` is actually not the correct name in mathlib).